### ISSUE TITLE
List - onOrder screenreader announcements

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7432]
### Expected Behavior
When interacting with the list onOrder button a screenreader user should have a clear sense of what happened when the interact with the up/down buttons.

### Actual Behavior
From the screenreader announcements it is difficult to determine what is happening when moving a list item up/down. It is also unclear where the focus lands after moving a list item.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/visualizations-list-order--order

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by jcfilben** [comment]
Do more research to evaluate what if anything we need to change to give more context

**Comment by britt6612** [comment]
Have a meeting on the calendar with Bill to run through for June 30th 2025

**Comment by britt6612** [comment]
Reviewed the List component `onOrder` with Bill. He found the component intuitive, providing the right amount of information and clear guidance for moving items within the list. Closing ticket.
